content/en/integrations/kubernetes.md

```diff
@@ -69,14 +70,32 @@ Note: these metrics are unavailable for Azure Kubernetes Service (AKS) at this p
 
 ## Collect container logs
 
-_Available for Agent versions >6.0_
+**Available for Agent >6.0**
 
-Two installations are possible:
+There are two ways to collect logs from containers running in Kubernetes:
 
-- On the node where the Agent is external to the Docker environment
-- Deployed with its containerized version in the Docker environment
+- Through the Docker socket.
+- Through the Kubernetes log files.
 
-Take advantage of DaemonSets to [automatically deploy the Datadog Agent on all your nodes][6]. Otherwise follow the [container log collection steps][7] to start collecting logs from all your containers.
+Datadog recommends using the Kubernetes log files approach when you are either not using Docker, or are using more than 10 containers per pod.
```

Comment:
Same as before, change "or are using more than 10 containers per pod" to "or are using more than 10 containers per node".